src/rebar3.erl

```diff
@@ -345,7 +345,13 @@ start_and_load_apps(Caller) ->
     ensure_running(public_key, Caller),
     ensure_running(ssl, Caller),
     inets:start(),
-    inets:start(httpc, [{profile, rebar}]).
+    inets:start(httpc, [{profile, rebar}]),
+    case ets:info(?REPOS_TABLE) of
+        undefined ->
+            ets:new(?REPOS_TABLE, [named_table, public]);
+        _ ->
+            ok
+    end.
```

Comment:
It is where registries are stored. It allows them to be loaded once and not carried around in the state.

I don't recall why I wanted it done this way instead of storing as a dict or map or even proplist in the state... It may have been simply because the way registries worked before was global and not passed in the state, but if that is the only reason it sounds like a bad reason.